src/OpenTelemetry.AutoInstrumentation/Configurations/ResourceConfigurator.cs

```diff
@@ -40,7 +41,8 @@ public static ResourceBuilder CreateResourceBuilder(IReadOnlyList<ResourceDetect
             resourceBuilder = enabledResourceDetector switch
             {
                 ResourceDetector.Container => Wrappers.AddContainerResourceDetector(resourceBuilder),
-                _ => resourceBuilder,
+                ResourceDetector.AzureAppService => Wrappers.AddAzureAppServiceResourceDetector(resourceBuilder),
```

Comment:
We could also include `AzureVMResourceDetector` it's a part of same package.

Ref: https://github.com/open-telemetry/opentelemetry-dotnet-contrib/tree/main/src/OpenTelemetry.ResourceDetectors.Azure#vm-resource-detector